example/spec/parallel/testAPI/granuleSpec.js

```diff
@@ -125,6 +125,7 @@ describe('The Granules API', () => {
       discoveredGranule = await getGranule({
         prefix,
         granuleId,
+        collectionId,
```

Comment:
These tests are updated to use the new endpoints. `getGranule` accepts an optional `collectionId`, which we're passing here.